server.go

```diff
@@ -95,6 +97,22 @@ func runMain() error {
 		return fmt.Errorf("Failed to initialize the smtp server: %v", err)
 	}
 
+	// get the default master extended public key
+	voteKey, err := stakepooldConnMan.DefaultAccountPubKey()
+	if err != nil {
+		return fmt.Errorf("Failed to get master extended public key for the \"defualt\" account: %v", err)
```

Comment:
typo: defualt

Reply:
In most cases, `error` strings should not begin with capital letters (starting with a type name that has a cap is one of the exceptions).